packages/tailwind/src/utils/css/populate-parents-for-node-tree.ts

```diff
@@ -0,0 +1,29 @@
+import { type CssNode, type List, type ListItem, walk } from 'css-tree';
+
+// This expands the definition for CssNode so that we can add a parent property to all nodes
```

Comment:
I'm not a big fan of this, we override an external dependency which we have no control of, so if they end up adding something that messes up with our definition we wouldn't know. have you explored other solutions that do not involve adding these properties?